### DISCUSSION TITLE
Report: Civilization VI

### DISCUSSION THREAD

**Discussion by P-IH** [canonical/steam-snap#175] [Game Reports]
Civilization VI is available with Linux, but since an update in december 2022, you will not get the full functinality with Linux. To achieve this you need to run the game in compatility mode.
Civilization VI do not start in compatility mode.
The installation works OK
Civilization starts in two steps, when you click start Civilization starts a 2K menu. This menu starts, but CIV VI itself do not start and you get back to your library view and are able to click start again. You will get the 2K Menu and your library view.
This is on a completely new installation.
On this computer I have earlier installed  Lunar and the Flathub version of Steam. This works OK with CIV VI.
I have also another computer with Ubuntu 22.04 and the deb installation of Steam, which also works OK with CIV VI

**Comment by P-IH** [comment]
Perhaps this is the same issue as #126.
My snap connection steam output
```
p-i@p-i-asus-b450:~$ snap connections steam
Interface               Plug                          Slot                                Notes
audio-playback          steam:audio-playback          :audio-playback                     -
audio-record            steam:audio-record            -                                   -
bluez                   steam:bluez                   -                                   -
content[gaming-mesa]    steam:gaming-mesa             gaming-graphics-core22:gaming-mesa  -
content[gtk-3-themes]   steam:gtk-3-themes            gtk-common-themes:gtk-3-themes      -
content[icon-themes]    steam:icon-themes             gtk-common-themes:icon-themes       -
content[sound-themes]   steam:sound-themes            gtk-common-themes:sound-themes      -
desktop                 steam:desktop                 :desktop                            -
desktop-legacy          steam:desktop-legacy          :desktop-legacy                     -
fuse-support            steam:fuse-support            -                                   -
gsettings               steam:gsettings               :gsettings                          -
hardware-observe        steam:hardware-observe        :hardware-observe                   manual
home                    steam:home                    :home                               -
joystick                steam:joystick                -                                   -
mount-observe           steam:mount-observe           -                                   -
network                 steam:network                 :network                            -
network-control         steam:network-control         -                                   -
opengl                  steam:opengl                  :opengl                             -
personal-files          steam:dot-local-share-steam   -                                   -
process-control         steam:process-control         -                                   -
removable-media         steam:removable-media         :removable-media                    -
screen-inhibit-control  steam:screen-inhibit-control  :screen-inhibit-control             -
shared-memory           steam:shmem                   :shared-memory                      -
steam-support           steam:steam-support           :steam-support                      -
system-observe          steam:system-observe          :system-observe                     manual
upower-observe          steam:upower-observe          :upower-observe                     -
wayland                 steam:wayland                 :wayland                            -
x11                     steam:x11                     :x11                                -
```

**Comment by ashuntu** [comment]
Could you edit your original post or post a reply with the output of running `snap run steam.report --no-submit` from a terminal? It may also be worth adding the terminal output from Steam (by running Steam via a terminal with `snap run steam` and then running the game in Steam).

**Comment by P-IH** [comment]
Ok, will do that tomorrow.

Den mån 20 mars 2023 kl 17:04 skrev Ash ***@***.***>:

> Could you edit your original post or post a reply with the output of
> running snap run steam.report --no-submit from a terminal? It may also be
> worth adding the terminal output from Steam (by running Steam via a
> terminal with snap run steam and then running the game in Steam).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/canonical/steam-snap/discussions/175#discussioncomment-5371427>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANBFNBQUN6FA2UHXKGTJE2DW5B5YJANCNFSM6AAAAAAWBGRW7I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Skickat från min Sony Xperia eller Ubuntu

**Reply by P-IH** [reply]
This is the result:
```
p-i@p-i-asus-b450:~$ snap run steam.report --no-submit
/usr/share/libdrm/amdgpu.ids: No such file or directory
os_release: 
    name:               "Ubuntu"
    version:            "23.04 (Lunar Lobster)"
snap_info: 
    steam_revision:     124
    snapd_revision:     18357
lspci: 
    0b:00.0:            Advanced Micro Devices, Inc. [AMD/ATI] Navi 10 [Radeon RX 5600 OEM/5600 XT / 5700/5700 XT] (rev c4)
glxinfo: 
    gpu:                NAVI10 (navi10, LLVM 15.0.7, DRM 3.49, 6.1.0-16-generic)
    gpu_version:        4.6 (Core Profile) Mesa 23.0.0 - kisak-mesa PPA
lscpu: 
    model_name:         AMD Ryzen 7 1700 Eight-Core Processor
xdg_current_desktop:    ubuntu:GNOME
desktop_session:        ubuntu
```

**Reply by P-IH** [reply]
This is the information when starting from the terminal.
file:///home/p-i/Documents/terminal-startofSteam-startofciv.txt
[terminal-startofSteam-startofciv.txt](https://github.com/canonical/steam-snap/files/11025837/terminal-startofSteam-startofciv.txt)

**Reply by P-IH** [reply]
In case you are interested, I'm able to start Steam in the terminal with an OK DEB version of Steam on another computer.

**Comment by ashuntu** [comment]
I added your information to the existing report for Civilization VI here: #124
